pulsar-broker/src/main/java/org/apache/pulsar/broker/service/BrokerService.java

```diff
@@ -3432,10 +3432,10 @@ private CompletableFuture<Boolean> isAllowAutoTopicCreationAsync(final TopicName
             return CompletableFuture.completedFuture(false);
         }
 
-        // ServiceUnitStateChannelImpl.TOPIC expects to be a non-partitioned-topic now.
+        // ExtensibleLoadManagerImpl.internal topics expects to be non-partitioned-topics now.
```

Comment:
Should we `checkBundleOwnership` in `handlePartitionMetadataRequest`?

Reply:
I think that can be a separate PR.

Here, I think we better explicitly say we don't allow topic auto-creation for these system topics, as we pre-create them.

Reply:
One thing we need to think about is that checking ownership there might cause some performance impact, as we will redirect to the owner brokers for every producer and consumer creation(and other topic related operations).